packages/dm-core-plugins/src/form/fields/ObjectField.tsx

```diff
@@ -51,7 +59,12 @@ const AddUncontained = (props: {
   )
 }
 
-const AddObject = (props: any) => {
+const AddObject = (props: {
+  type: string
+  namePath: string
+  onAdd: () => void
+  idReference?: string
```

Comment:
is idReference optional? 
From the usage inside the component it might seem like it's required

Reply:
It is not optional but it was set that way before #290. Will fix